content/html/concepts/semantic-html/terms/ruby/ruby.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '<ruby>'
+Description: 'Represents small annotations usually used for showing pronunciation of Japanese and East Asian characters.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<ruby>`** tag is a semantic HTML element that represents small annotations usually used for showing pronunciation of Japanese and East Asian characters. You can use `<ruby>` together with `<rt>` /or `<rp>`. The `<ruby>` entails one or more chacters which require an explanation or pronunciation, and an `<rt>` element that provides that information, and an optional `<rp>` element which ascertains what to show to browsers that lack support for annotations.
```

Comment:
Remove pronouns (I we you) and keep language definitive, I.e. Use <this> to accomplish <that>.